Apsitvarkom.Api/Middleware/LogRequestStatisticsMiddleware.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Diagnostics;
+
+namespace Apsitvarkom.Api.Middleware
+{
+    public class LogRequestStatisticsMiddleware
+    {
+        private readonly RequestDelegate _next;
+        private int _currentRequest = 1;
+
+        public LogRequestStatisticsMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public async Task InvokeAsync(HttpContext context)
+        {
+            var requestId = _currentRequest++;
```

Comment:
I am afraid this may cause race conditions (using a primitive `int`). AFAIK, asp.net core handles requests concurrently, so bad things might happen if two threads try to access the same memory block at the same time. Using `Interlocked` should help.